integrationTesting/tests/organize/campaigns/list/list.spec.ts

```diff
@@ -22,12 +22,13 @@ test.describe('Campaigns list page ', () => {
       WelcomeNewMembers,
     ]);
 
-    await page.goto(appUri + '/organize/1/projects');
+    const campaignCards = page.locator('data-testid=campaign-card');
+    await Promise.all([
+      page.goto(appUri + '/organize/1/projects'),
+      campaignCards.first().waitFor({ state: 'visible' }),
+    ]);
```

Comment:
Is this really necessary though?

I've used the `Promise.all([ someAction, someWait ])` construct a lot for cases where the two can happen simultaneously, e.g. where `someAction` is a click or a navigation, and `someWait` is, for example, an HTTP request that gets made as a result of that click. In those cases, the request can sometimes happen _while_ the click evaluates, and when that happens the test would stall if we first waited for the click, and then waited for the request (which by then had already happened and won't happen again).

But in this situation, and others in this PR, I feel like the `campaignCards.first().waitFor()` can always happen _after_ the navigation. Isn't that right?

Here's what the documentation on `waitFor` says:

>Returns when element specified by locator satisfies the state option.
>
>If target element already satisfies the condition, the method returns immediately. Otherwise, waits for up to timeout milliseconds until the condition is met.

That makes me think that if `await page.goto(appUri + '/organize/1/projects')` finishes "late", once the campaign cards have already shown up, that won't be a problem because `waitFor()` will just return immediately and the test can proceed.

If I'm right, it means we don't have to use the `Promise.all()` construct here and I think that would be better as it would make the test more readable and "sequential".